app/_hub/kong-inc/http-log/_index.md

```diff
@@ -96,15 +96,10 @@ params:
       default: empty table
       datatype: array of string elements
       description: |
-        An optional table of headers added to the HTTP message to the upstream server. The following
-        headers are not allowed: `Host`, `Content-Length`, `Content-Type`.
+        An optional table of headers (value indexed by header name) added to the HTTP message to the
+        upstream server. The following headers are not allowed: `Host`, `Content-Length`, `Content-Type`.
 
-        **Note:** This parameter is only available for versions
-        2.3.x and later.
-  extra: |
-    **NOTE:** If the `config.http_endpoint` contains a username and password (for example,
-    `http://bob:password@example.com/logs`), then Kong Gateway automatically includes
-    a basic-auth `Authorization` header in the log requests.
+        **Note:** Before version 3.0.0 the values were arrays of strings (multiple values per header name)
```

Comment:
@Tieske Do you have an example of what the new format should be?

Reply:
see the "custom headers" section below. That is the example we currently have, but is broken in the current version. In 3.0 that example will work.

Adding a commit to update the 2.1 version to the proper format with multiple values.